src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,61 @@
+package mate.jdbc.model;
+
+public class Driver {
+    private long id;
+    private String name;
+    private String licenceNumber;
+
+    public Driver(String name, String licenceNumber) {
+        this.name = name;
+        this.licenceNumber = licenceNumber;
+    }
+
+    public Driver(long id, String name, String licenceNumber) {
+        this.id = id;
+        this.name = name;
+        this.licenceNumber = licenceNumber;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getLicenceNumber() {
+        return licenceNumber;
+    }
+
+    public void setLicenceNumber(String licenceNumber) {
+        this.licenceNumber = licenceNumber;
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();

```

Comment:
why such implementation of `equals` and `hashCode`? I mean implementation with using `super`.